app/Book.php

```diff
@@ -18,6 +18,30 @@ public function getUrl($path = false)
         return baseUrl('/books/' . urlencode($this->slug));
     }
 
+    public function getBookCover()
+    {
+        $default = baseUrl('/default.png');
+        $image = $this->image;
+        if ($image === 0 || $image === '0' || $image === null) 
+            return $default;
+        try {
+            $cover = $this->cover ? baseUrl($this->cover->getThumb(120, 192, false)) : $default;
```

Comment:
- It's better to take width and height parameters instead of hard coding directly, so it will help us in feature to retrieve book cover in whatever size we want.